manifests/profile/www_lib/vhosts/www_lib.pp

```diff
@@ -0,0 +1,76 @@
+
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::www_lib::vhosts::www_lib
+#
+# www.lib.umich.edu virtual host
+#
+# @example
+#   include nebula::profile::www_lib::vhosts::www_lib
+class nebula::profile::www_lib::vhosts::www_lib (
+  String $prefix,
+  String $domain,
+  String $vhost_root,
+  String $ssl_cn = 'www.lib.umich.edu'
+) {
+
+  nebula::apache::www_lib_vhost { 'www.lib-ssl':
+    servername                    => "${prefix}www.${domain}",
+    ssl                           => true,
+    vhost_root                    => $vhost_root,
```

Comment:
get rid of this and add scriptalias, see comment above

Reply:
or maybe just make it a local variable here rather than passing through to `www_lib::vhost`